### DISCUSSION TITLE
is MapboxGL.offlineManager.resetDatabase() not working ?

### DISCUSSION THREAD

**Discussion by NortacYWang** [rnmapbox/maps#2553] [Q&A]
Hello, i am trying to use resetDatabase() functions, and this is my code: 

`
      <Button

        title="clear offline database"

        onPress={async () => {

          await MapboxGL.offlineManager.resetDatabase();

          console.log('offlinedata', offlineData);

        }}

      />
`

and upon I click it, the react-native debugger returns : 
![image](https://user-images.githubusercontent.com/96532687/213465555-50a30c8d-60d2-4d9d-8c55-f3976f676e1d.png)

is it a bug or I did something wrong ? 

Thanks

**Comment by sampathkumarch** [comment]
@NortacYWang, same issue facing unable to remove the packs, if you find the solution  can you help in this point

**Comment by sampathkumarch** [comment]
@NortacYWang
 hi , can you get the solution , am also facing same issue , can you help me that point

**Reply by NortacYWang** [reply]
Hello, @sampathkumarch, unfortunately no,  we changed our map and used google map instead

**Comment by sampathkumarch** [comment]
@NortacYWang, thanks for update, can help me how to integrate the google maps in react native app, with offline maps capabilities, 
actually, I want use as offlinemap, with help of offlinemap can locate the my GPS location tracking, how to achieve this functionality  please guide me.

**Reply by NortacYWang** [reply]
Hi, @sampathkumarch , check this: https://www.npmjs.com/package/react-native-maps   you can use localTile / urlTile to display local xyz format map(you may need to convert your map to xyz format by using other software).  And be careful it may not work on RN debug mode.
